### PR TITLE
Feature/idva5 837 option 2 by a person

### PR DESCRIPTION
**Pull request by hemerem-ch**
-build document checked page 
-added service to save options
-added tests

### CODE REVIEW

File: src/views/identity-documents-group-2/identity-documents-group-2.njk [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+                text: i18n.ukFirearmsLicence,
+                attributes: {
+                    "data-event-id": "ukFirearmsLicence"
+                }
+            },
+            {
+                value: "photoIdPrado",
+                text: i18n.photoIdPrado,
+                hint: {
+                        text: "Such as a National Identity Card (Pakistan), crew member certificate (South Africa), or Permanent Resident Card (USA)."
```
Comment (swalke-ch) [inline src/views/identity-documents-group-2/identity-documents-group-2.njk:124]: Can this be pulled from locales please?

**Review by sardhani-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/validations/identityDocumentsGroup2.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { body } from "express-validator";
+
+export const identityDocsGroup2Validator = [
+    body("documentsGroup2A", "identityDocumentsGroup1Empty").notEmpty(),
+    body("documentsGroup2B", "identityDocumentsGroup1Empty").notEmpty()
```
Comment (ttingle-ch) [inline src/validations/identityDocumentsGroup2.ts:5]: User needs to select 2 documents but with at least 1 from group A but can choose 2 from group A and 0 from group B

**Review by sampankumar — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Pull request merged**